tests/queries/0_stateless/03198_table_function_directory_path.sql

```diff
@@ -0,0 +1,13 @@
+-- Tags: no-parallel
+
+INSERT INTO FUNCTION file('data_03198_table_function_directory_path/1.csv', 'csv') SELECT '1.csv' SETTINGS engine_file_truncate_on_insert=1;
+INSERT INTO FUNCTION file('data_03198_table_function_directory_path/2.csv', 'csv') SELECT '2.csv' SETTINGS engine_file_truncate_on_insert=1;
```

Comment:
It would be slightly better to use unique paths for each test run (possible in .sh tests), but `no-parallel` and `engine_file_truncate_on_insert` works too